src/consensus/consensus.ml

```diff
@@ -0,0 +1,16 @@
+include State
+include Effect
```

Comment:
why not swap State and Effect that way you wont need to define type t = state

Reply:
good one, but please address on the latest PR as it's a nitpick